docs/en/advanced/Client-Logging.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: global
+title: Client Logging
+nickname: Client Logging
+group: Advanced
+priority: 1
+---
+
+* Table of Contents
+{:toc}
+
+## Introduction
+
+This page summarizes how to find client-specific logs and modify logging levels in order to debug
+client-side issues with your Alluxio installation.
+
+## Log Location
+
+Depending on which compute framework you are using, the location of the logs can vary. We
+recommend looking at the specific documentation for your compute framework to determine where to
+find alluxio client logs.
+
+Here are links to compute frameworks with information where logs may be found and how to configure
+
+- [Apache Hadoop]({{ site.baseurl }}{% link en/compute/Compute-Hadoop-MapReduce.md %}#logging-configuration)
+- [Apache HBase]({{ site.baseurl }}{% link en/compute/Compute-HBase.md %}#logging-configuration)
+- [Apache Hive]({{ site.baseurl }}{% link en/compute/Compute-Hive.md %}#logging-configuration)
+- [Apache Spark]({{ site.baseurl }}{% link en/compute/Compute-Spark.md %}#logging-configuration)
+
+However, by setting the `alluxio.logs.dir` property you can modify where the logs are stored.
```

Comment:
Is this true? I think `alluxio.logs.dir` is only applicable when using the `log4j.properties` file that comes in the Alluxio distribution.

Reply:
whoops, nope. I meant to remove that paragraph.